integration-cli/docker_cli_run_test.go

```diff
@@ -61,6 +61,14 @@ func (s *DockerSuite) TestRunWithoutMemoryswapLimit(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestRunWithSwappiness(c *check.C) {
+	runCmd := exec.Command(dockerBinary, "run", "--memory-swappiness", "0", "busybox", "true")
```

Comment:
some extra tests here showing that it can't be set less than 0 or greater than 100 would be fantastic.
Also probably demonstrating the default value is set to 60 would be great (to make sure this doesn't change by accident).